src/main/java/com/agilethought/internship/sso/services/ServiceApplicationimpl.java

```diff
@@ -20,6 +22,9 @@
 	
 	@Override
 	public UserId createUser(User user) {
+		BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();

```

Comment:
Try to avoid the creation of multiple objects, you can avoid this  passwordEncoder  creation adding a bean in the SecurityConfiguration class like below: 

@Bean
	protected PasswordEncoder passwordEncoder() {
		return new BCryptPasswordEncoder();
	}

An finally here in the ServiceApplicationimpl you only will call the bean using the @Autowired annotation like below :

@Autowired
	private PasswordEncoder passwordEncoder;


And you will be able to use passwordEncoder.encode(password) using the same bean in any part of the application.

Reply:
thank you, done